src/components/Settings/SettingsLeague.jsx

```diff
@@ -5,6 +5,30 @@ import {
 } from '@material-ui/core';
 import '../../css/SettingsLeague.css';
 
+const inputsForm = [
+  {
```

Comment:
reformat these to have each key/value pain on its own line

Reply:
I'll have my changes for the next PR